tests/integration/cards.test.js

```diff
@@ -361,7 +361,7 @@ describe('CARDS ROUTES - POST /cards/{_id}/answer', () => {
       expect(response.statusCode).toBe(404);
     });
 
-    it('should return 403 if too many answers', async () => {
+    it('should return 403 if question answer with already max answers', async () => {
```

Comment:
peut-etre que tu peux merge tes trois` it should return 403 if too man`y en un seul avec un tableau contenant les trois templates puis 
```
templates.forEach((template) => {
      it(`should return 403 if ${template.name} with already max answers`, async () => {
```

Reply:
done